src/main/java/cn/nukkit/item/Item.java

```diff
@@ -399,7 +399,10 @@ private static void initCreativeItems() {
         // Creative inventory for 407+
         for (Map map : new Config(Config.YAML).loadFromStream(Server.class.getClassLoader().getResourceAsStream("creativeitems407.json")).getMapList("items")) {
             try {
-                addCreativeItem(v1_16_0, fromJson(map));
+                Item item = fromJson(map);
+                Item newItem = new Item(item.getId(), item.getDamage(), item.getCount());
+                newItem.setCompoundTag(item.getCompoundTag());
+                addCreativeItem(v1_16_0, newItem);
```

Comment:
This can fix the 1.16.200 problem, and it has no effect on the 1.16.0 version test